driver/mux.go

```diff
@@ -63,16 +66,23 @@ func (m *Mux) run() {
 	}
 }
 
-func NewMuxOutput(ctx *proc.Context, parents []proc.Proc) *MuxOutput {
+func NewMuxOutput(ctx *proc.Context, parents []proc.Proc, scanner *scanner.Scanner) *MuxOutput {
```

Comment:
Rather than add a `*scanner.Scanner` to MuxOutput, we could just say it takes a `interface{Stats() api.ScannerStats}`, if this is preferred. And/or rename MuxOutput to something like Flowgraph or Cradle to enshrine it as the thing that holds a running flowgraph.